eventuate-messaging-kafka-basic-consumer/src/test/resources/application.properties

```diff
@@ -2,7 +2,7 @@ eventuate.local.kafka.consumer.properties.session.timeout.ms=30000
 eventuate.local.kafka.consumer.properties.key.serializer=org.apache.kafka.common.serialization.StringSerializer
 
 eventuate.local.kafka.producer.properties.buffer.memory=1000000
-eventuate.local.kafka.producer.properties.value.serializer=org.apache.kafka.common.serialization.StringSerializer
+eventuate.local.kafka.producer.properties.value.serializer=org.apache.kafka.common.serialization.ByteArraySerializer
```

Comment:
Are you sure that this should this be configurable?

Reply:
It is just a test for property parsing. We can add different kafka configs, including this.
To make it not configurable, we need to add direct blocking (checks with exceptions) for such properties.